soroban-sdk-auth/src/public_types.rs

```diff
@@ -7,29 +7,29 @@ pub struct KeyedEd25519Signature {
     pub signature: BytesN<64>,
 }
 
-pub type AccountAuthorization = Vec<KeyedEd25519Signature>;
+pub type AccountSignatures = Vec<KeyedEd25519Signature>;
 
 #[derive(Clone)]
 #[contracttype]
-pub struct KeyedAccountAuthorization {
+pub struct KeyedAccountSignatures {
     pub account_id: BytesN<32>,
-    pub signatures: AccountAuthorization,
+    pub signatures: AccountSignatures,
```

Comment:
Do we use `AccountSignatures` independent of `KeyedAccountSignatures`? If no, I think we can remove the type alias and rename the struct to `AccountSignature`.

Reply:
Done